run_eval.sh

```diff
@@ -0,0 +1,9 @@
+MODEL_PATH=/cb/cold/checkpoints/llms/HF/Meta-Llama-3-8B-Instruct #Phi-3-mini-128k-instruct
```

Comment:
I'd either make this script generic with paths to model and python removed, or skip adding it entirely.